app/helpers/react_components/student/nudge.rb

```diff
@@ -13,7 +13,7 @@ def to_s
           discussions: discussions,
           request: request,
           exercise_type: exercise_type,
-          iterations: solution.iterations.map { |iteration| SerializeIteration.(iteration) },
+          iterations: solution.iterations.order(idx: :desc).map { |iteration| SerializeIteration.(iteration) },
```

Comment:
This matches the behavior for the complete button on the right hand side. See https://github.com/exercism/website/blob/main/app/helpers/react_components/student/complete_exercise_button.rb#L11

Reply:
Nice :)